kaiax/builder/impl/api.go

```diff
@@ -39,4 +47,37 @@ func NewBuilderAPI(b *BuilderModule) *BuilderAPI {
 	return &BuilderAPI{b}
 }
 
-// TODO: implement SendRawTransactions
+func (s *BuilderAPI) SendRawTransactions(ctx context.Context, inputs []hexutil.Bytes) ([]common.Hash, error) {
```

Comment:
It's a question. As far as I understand, if a user sends the [approvalTx, swapTx], then the first attempt to promote `approvalTx` will fail (`IsReady == false`) since the tx pool doesn't know about the swapTx but will be retained in the queue and promoted with `swapTx` insertion. Is it right?

Reply:
Yes, right.